test/nn/encoder/test_stype_encoder.py

```diff
@@ -44,10 +45,12 @@ def test_categorical_feature_encoder(encoder_cls_kwargs):
         stype=stype.categorical,
         **encoder_cls_kwargs[1],
     )
-    feat_cat = tensor_frame.feat_dict[stype.categorical]
+    feat_cat = tensor_frame.feat_dict[stype.categorical].clone()
```

Comment:
This does not really test anything since the `dataset` does not contain `nan` in the first place. Same for others.

Reply:
I am testing the in-place modification here. So `feat_cat` should be the same value has the `tensor_frame.feat_dict[stype.categorical]` after being passed to encoders

Reply:
Yes I understand. My point is that the original data does not contain any NaN, so there is likely no in-place modification at all. What's interesting to test is when the data contains NaN.

Reply:
Clarified offline that this is testing there there are no-inplace modifications without nans